lib/tasks/maxminddb.rake

```diff
@@ -34,6 +34,7 @@ def copy_maxmind(from_path, to_path)
     from = File.join(from_path, "#{name}.mmdb")
     to = File.join(to_path, "#{name}.mmdb")
     FileUtils.cp(from, to, preserve: true)
```

Comment:
I might be wrong but I think `preserve: true` preserves file times which sounds like what you don't want?

Reply:
Yes but we still want to preserve the owner and group of the file.

Reply:
> fileutils.rb:863
> # - <tt>preserve: true</tt> - preserves file times.

Reply:
Wah, that `copy_metadata` and comment is a bit confusing.